packages/app/src/cli/services/app/config/link.ts

```diff
@@ -123,7 +124,7 @@ async function loadConfigurationFileName(
     return configurationFileNames.app
   }
 
-  const configName = await selectConfigName(options.directory, remoteApp.title)
+  const configName = await selectConfigName(localApp.directory ?? options.directory, remoteApp.title)
```

Comment:
it's a different method. `directory` variable is not set and in this case it does not make sense to declare it as it's only used in once place

Reply:
Ah, I thought it was the same one.

Anyway, I guess we also want to ensure it doesn't fail when localApp is undefined (by adding the `?` operator), and use the one from the directory when it's empty (`||` instead of `??`). 
```suggestion
  const configName = await selectConfigName(localApp?.directory || options.directory, remoteApp.title)
```

Reply:
it does not make sense to declare `localApp` as optional in this method as it's only called once and it always has a value. I removed that